src/carousel/carousel.ts

```diff
@@ -83,7 +84,10 @@ export class NgbSlide {
     <ol class="carousel-indicators" [class.visually-hidden]="!showNavigationIndicators" role="tablist">
       <li *ngFor="let slide of slides" [class.active]="slide.id === activeId"
           role="tab" [attr.aria-labelledby]="'slide-' + slide.id" [attr.aria-controls]="'slide-' + slide.id"
+          [attr.data-bs-target]="id"
+          [attr.data-bs-to]="'slide-' + slide.id"
```

Comment:
What is the point of adding this attribute? If you want to do as bootstrap, there is a `data-bs-slide-to` attribute and it contains only the slide number, but I don't see any `data-bs-to` attribute.
cf (again) https://getbootstrap.com/docs/5.1/components/carousel/#with-indicators